e2e_tests/docker_image/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM ubuntu:18.04
+
+ENV PKG_CONFIG_PATH /usr/local/lib/pkgconfig
+
+RUN apt update
+RUN apt install -y autoconf-archive libcmocka0 libcmocka-dev procps
+RUN apt install -y iproute2 build-essential git pkg-config gcc libtool automake libssl-dev uthash-dev doxygen libjson-c-dev
+RUN apt install -y --fix-missing wget python3 cmake clang
+RUN apt install -y gcc-multilib
+RUN apt install -y gcc-arm-linux-gnueabihf
+RUN apt install -y gcc-aarch64-linux-gnu
+RUN apt install -y libini-config-dev libcurl4-openssl-dev tpm2-tools curl libgcc1
+
+WORKDIR /tmp
+
+# Download and install TSS 2.0
+RUN git clone https://github.com/tpm2-software/tpm2-tss.git --branch 2.3.3
+RUN cd tpm2-tss \
+	&& ./bootstrap \
+	&& ./configure \
+	&& make -j$(nproc) \
+	&& make install \
+	&& ldconfig
+
+RUN rm -rf tpm2-tss
```

Comment:
Would it work to use the [Ubuntu package](https://packages.ubuntu.com/source/bionic/tpm2-tss) here instead?

Reply:
Hmm, I only searched for what's directly accessible from `apt` (and it doesn't appear there), but maybe if I add the correct registry... Will try.

Reply:
Oh... The thing you linked is a source package - it just contains the source that you build from. Looks like those sources were used to compile [`libsapi`](https://packages.ubuntu.com/bionic/libsapi-dev), so no `esapi`. They also mention on their [Install](https://github.com/tpm2-software/tpm2-tss/blob/master/INSTALL.md) page that there's a package available for Fedora, but there's no mention of Ubuntu/Debian.

I was wondering if we should switch to testing on Fedora, I think there are a few dependencies that we could get for free.

Reply:
Worth to check but it's fine if the size is not going to be a lot smaller actually..